stdlib/universe/group_nulls_test.flux

```diff
@@ -1,114 +1,72 @@
-,,1,2018-05-22T19:54:06Z,15205499,io_time,diskio,host1,
-,,1,2018-05-22T19:54:16Z,,io_time,diskio,host1,
-,,1,2018-05-22T19:53:26Z,,io_time,diskio,host1,disk0
-,,1,2018-05-22T19:53:36Z,15204894,io_time,diskio,host1,disk0
-,,1,2018-05-22T19:53:46Z,15205102,io_time,diskio,host1,disk0
-,,1,2018-05-22T19:53:56Z,,io_time,diskio,host1,disk0
-,,1,2018-05-22T19:54:06Z,15205499,io_time,diskio,host1,disk0
-,,1,,15205755,io_time,diskio,host1,disk0
-,,1,2018-05-22T19:53:26Z,15204688,io_time,diskio,host1,disk1
-,,1,,,io_time,diskio,host1,disk1
-,,1,2018-05-22T19:53:46Z,,io_time,diskio,host1,disk1
-,,1,2018-05-22T19:53:56Z,15205226,io_time,diskio,host1,disk1
-,,1,2018-05-22T19:54:06Z,15205499,io_time,diskio,host1,disk1
-,,1,2018-05-22T19:54:16Z,15205755,io_time,diskio,host1,disk1
-
-#datatype,string,long,dateTime:RFC3339,long,string,string,string
-#group,false,false,false,false,false,false,true
-#default,_result,,,,,,
-,result,table,_time,_value,_field,_measurement,host
-,,2,,15204688,io_time,diskio,host2
-,,2,2018-05-22T19:53:36Z,,io_time,diskio,host2
-,,2,2018-05-22T19:53:46Z,15205102,io_time,diskio,host2
-,,2,2018-05-22T19:53:56Z,15205226,io_time,diskio,host2
-,,2,2018-05-22T19:54:06Z,15205499,io_time,diskio,host2
-,,2,2018-05-22T19:54:16Z,,io_time,diskio,host2
-"
 
 testcase group_nulls {
+    // FIXME: in C2/OSS since the group key incorrectly empty for the case where `t0` is null.
+    // Remove skip tag after https://github.com/influxdata/flux/issues/5178
     option testing.tags = ["skip"]
 
+    input =
+        array.from(
+            rows: [
+                {
+                    _time: 2022-09-07T00:00:01Z,
+                    _measurement: "m0",
+                    _field: "f0",
+                    _value: 1,
+                    t0: "a",
+                },
+                {
+                    _time: 2022-09-07T00:00:02Z,
+                    _measurement: "m0",
+                    _field: "f0",
+                    _value: 2,
+                    t0: "a",
+                },
+                {
+                    _time: 2022-09-07T00:00:03Z,
+                    _measurement: "m0",
+                    _field: "f0",
+                    _value: 3,
+                    t0: "a",
+                },
+                {
+                    _time: 2022-09-07T00:00:01Z,
+                    _measurement: "m0",
+                    _field: "f0",
+                    _value: 1,
+                    t0: debug.null(type: "string"),
+                },
+                {
+                    _time: 2022-09-07T00:00:02Z,
+                    _measurement: "m0",
+                    _field: "f0",
+                    _value: 2,
+                    t0: debug.null(type: "string"),
+                },
+                {
+                    _time: 2022-09-07T00:00:03Z,
+                    _measurement: "m0",
+                    _field: "f0",
+                    _value: 3,
+                    t0: debug.null(type: "string"),
```

Comment:
When we have nulls in the input we generally can't run it against C2/OSS as the nulls will be lost when written. So I think we can just skip `testing.load` and close https://github.com/influxdata/flux/issues/5178 ?

Reply:
I think nulls in tags are expected to work here since tags can be added over time.

Several of the issues with the original test data fall into this trap you highlight (nulls for value, time, measurement all get dropped during ingest), but in the case of the tag I think we still want this coverage.